packages/parrot-devtools/src/app/components/ScenarioSelector.jsx

```diff
@@ -35,6 +35,7 @@ const ScenarioSelector = ({
           <span className="alert-icon dls-icon-warning icon-lg" />
           <h1 className="heading-4 alert-header">Failed to Load Scenarios</h1>
           <p className="alert-subtitle">
+            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid, jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
             Try <a onClick={loadScenarios}>refreshing</a> or updating the middleware URL in
```

Comment:
Change this to `button` to so we don't have to write all these exceptions to eslint?

Reply:
updated 824f96d